NLog.Windows.Forms/ToolStripItemTarget.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Forms;
+using NLog.Common;
+using NLog.Config;
+using NLog.Targets;
+
+namespace NLog.Windows.Forms
+{
+    /// <summary>
+    /// Logs text to Windows.Forms.ToolStripItem.Text property control of specified Name.
+    /// </summary>
+    /// <example>
+    /// <p>
+    /// To set up the target in the <a href="config.html">configuration file</a>, 
+    /// use the following syntax:
+    /// </p>
+    /// <code lang="XML" source="examples/targets/Configuration File/ToolStripItem/NLog.config" />
+    /// <p>
+    /// The result is:
+    /// </p>
+    /// <img src="examples/targets/Screenshots/ToolStripItem/ToolStripItem.gif" />
+    /// <p>
+    /// To set up the log target programmatically similar to above use code like this:
+    /// </p>
+    /// <code lang="C#" source="examples/targets/Configuration API/ToolStripItem/Form1.cs" />,
+    /// </example>
+    [Target("ToolStripItem")]
+    public sealed class ToolStripItemTarget : TargetWithLayout
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ToolStripItemTarget" /> class.
+        /// </summary>
+        /// <remarks>
+        /// The default value of the layout is: <code>${longdate}|${level:uppercase=true}|${logger}|${message}</code>
+        /// </remarks>
+        public ToolStripItemTarget()
+        {
+        }
+
+        private delegate void DelSendTheMessageToFormControl(ToolStripItem control, string logMessage);
+
+        /// <summary>
+        /// Gets or sets the name of the ToolStripItem to which NLog will log write log text.
+        /// </summary>
+        /// <docgen category='Form Options' order='10' />
+        [RequiredParameter]
+        public string ItemName { get; set; }
```

Comment:
maybe nice to make all the properties layoutable?

Reply:
I'm not exactly sure how you mean layoutable. Is there an example of a layoutable property in the project I can use an an example to understand what you mean?

Reply:
Use the type `Layout` instead of `string`, and use .Render(logEvent).